code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -331,3 +331,18 @@
 	desc = "You sit in this. Either by will or force. Looks REALLY uncomfortable."
 	icon_state = "chairold"
 	item_chair = null
+
+/obj/structure/chair/sofa
+	name = "old ratty sofa"
+	icon_state = "sofaconnectors"
+	item_chair = null
+	anchored = 1
```

Comment:
`TRUE`

Reply:
things that happen if you just copy paste...